src/ontology/mondo-edit.obo

```diff
@@ -210887,9 +210887,6 @@ xref: UMLS:C0546264 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source=
 is_a: MONDO:0002921 {source="OMIM:255310"} ! congenital structural myopathy
 is_a: MONDO:0005336 {source="MONDO:Redundant", source="NCIT:C120046"} ! myopathy
 is_a: MONDO:0019952 {source="DOID:0080102"} ! congenital myopathy
-is_a: MONDO:0100108 {source="https://clinicalgenome.org/docs/clingen-cmp-ep-acgs-collaboration/"} ! TPM3-related myopathy
-is_a: MONDO:0100150 {source="https://clinicalgenome.org/affiliation/40031/"} ! RYR1-related myopathy
-is_a: MONDO:0100196 {source="PMID:28881016", source="https://clinicalgenome.org/affiliation/40031/", source="https://github.com/monarch-initiative/mondo/issues/1922"} ! TPM2-related myopathy
```

Comment:
@katiermullen, these should not simply be deleted. An ["excluded subclass"[(https://mondo.readthedocs.io/en/latest/editors-guide/g-logical-axioms/#excluded-subclassof:~:text=non%2Dlogical)%20axiom.-,Excluded%20subclassOf,%C2%B6,-Rather%20than%20removing) annotation should be added.